packages/blade/src/components/Typography/BaseText/getBaseTextStyles.ts

```diff
@@ -35,6 +37,18 @@ const getBaseTextStyles = ({
       };
     }
   }
+  if(shouldTruncate !== undefined) {
```

Comment:
We have `truncateAfterLines={1}` prop on Text which is expected to do the similar thing. Any idea why that one is not working? can that prop be used instead of new prop?
https://codesandbox.io/s/7gtqwk?file=/App.tsx&utm_medium=sandpack